src/main/java/org/folio/search/service/KafkaConstants.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.search.service;
+
+public class KafkaConstants {
+  public static final String EVENT_LISTENER_ID = "mod-search-events-listener";
```

Comment:
We can add private constructor just to remove code smell. 
And maybe we should move constants to utils package. Out of services?
```suggestion
  private KafkaConstants() {}
  public static final String EVENT_LISTENER_ID = "mod-search-events-listener";
```

Reply:
Fixed as suggested.